src/views/admin/PersonDetailsView.vue

```diff
@@ -262,6 +289,19 @@
   const alertButtonAction: ComputedRef<() => void> = computed(() => {
     return personenkontextStore.errorCode === 'PERSON_NOT_FOUND' ? navigateToPersonTable : (): void => router.go(0);
   });
+
+  const alertButtonTextKopers: ComputedRef<string> = computed(() => {
+    return personStore.errorCode === 'PERSONALNUMMER_NICHT_EINDEUTIG'
+      ? t('admin.person.backToForm')
+      : t('nav.backToList');
+  });
+
+  const alertButtonActionKopers: ComputedRef<() => void> = computed(() => {
+    return personStore.errorCode === 'PERSONALNUMMER_NICHT_EINDEUTIG'
+      ? navigateBackToKopersForm
+      : navigateToPersonTable;
+  });
```

Comment:
This specifically checks for duplicateed  Kopers error because we want to redirect the admin through the SPSH-Alert to the form if that's case in order to correct his Input. Every other error will also use this but then default to  t('nav.backToList'); and  navigateToPersonTable;